fbchat/client.py

```diff
@@ -88,11 +88,6 @@ def __init__(self, email, password, debug=True, user_agent=None):
     def _console(self, msg):
         if self.debug: print(msg)
 
-    def _setttstamp(self):
```

Comment:
I seem to recall this was also separate in the j JavaScript version (to be verified). If it's not, I don't mind moving the function

-----Original Message-----
From: "Sid Kapur" <notifications@github.com>
Sent: ‎07/‎04/‎2016 17:49
To: "carpedm20/fbchat" <fbchat@noreply.github.com>
Cc: "Pidgey" <pieterjan.moreels@gmail.com>
Subject: Re: [carpedm20/fbchat] Remove unnecessary global state (#34)

In fbchat/client.py:
> @@ -88,11 +88,6 @@ def __init__(self, email, password, debug=True, user_agent=None):
>      def _console(self, msg):
>          if self.debug: print(msg)
>  
> -    def _setttstamp(self):
Sorry, my earlier message was vague.
My main concern with this function is that it reads and writes to global state for no reason. It being a separate function is not my biggest concern.
As a secondary concern, however, if you would like to keep it a function, I suggest moving the function inside the _generatePayload function (I believe this is what I did in the PR), so that we are not exposing more functions than is necessary. This makes it easier for a newbie to read and understand your code, because they can easily tell that the function is only used inside _generatePayload.
—
You are receiving this because you commented.
Reply to this email directly or view it on GitHub